### PR TITLE
Connect Bookstore to API

### PR DESCRIPTION
**Pull request by braincee**
_**In this pull request, the following requirements were implemented:**_

- I added redux-thunk to your project.
- I used `axios` for making API calls.
- I fetched the book list from Bookstore API on the Books page load.
- I added an Action Creator, that returns a function.

- I dispatched another action that will pass the API response to the reducer and update the state.
- When refactor is done,  add book and remove book features that persist on changes in the server.

### CODE REVIEW

File: src/components/AddBook.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  return (
+    <form>
+      <input
+        type="text"
+        placeholder="Add Title..."
+        name="title"
+        value={book.title}
+        className="addtitle"
+        onChange={addTitle}
+      />
```
[the remark this answers is not in the document]
Comment (braincee) [reply src/components/AddBook.js:45]: @chaw-bot please the input field works perfectly and I am able to add and remove books. When page is refreshed the books added persists on the page.

[the remark this answers is not in the document]
Comment (braincee) [reply src/components/AddBook.js:45]: I have added three screenshots to my README file showing title and author input fields being displayed in the UI. I added four books in the first screenshot, deleted two books in the second screenshot, and added extra three books in the third screenshot.

**Review by Kingstalux — APPROVED** [review]
Hi @braincee,
# Status: Approved ✔️ ✔️ ✔️ 
 
Gu2dpSp0qee4C710M5kVsWN425J3-vv2w52by6

## Highlights 👏 
✅ App has the correct functionality 🔥 
✅ Books can be added and removed successfully 👍 
✅ Books are displayed with their title and author 👏 
✅ Good PR description 👌 

Good job so far 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by braincee** [comment]
Thanks very much, @chaw-bot and @Kingstalux 

the-office-bow

**Pull request merged**